test/e2e/framework/service/node.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package service
```

Comment:
It's not exactly clear why we have a `node.go` file in package `service`. I think it's worthwhile to have a package comment saying it's for testing backend nodes for services with `Type=LoadBalancer`.

Reply:
Maybe in a follow-up we should actually move this to test/e2e/framework/node, cc @jiatongw

Reply:
except `PodNodePairs() `, both of the other two methods are moved to `node` package in an opening PR #79879

Reply:
I think I can put `PodNodePairs()` in `node` package too and we can avoid a new file node.go here
@draveness @andrewsykim thoughts?

Reply:
Sounds good, but could we do this in #79879. Otherwise, I have to wait for that PR get merged. 🙂

Reply:
I think it's save to delete this file and revert the relate references. I've moved all 3 methods to `framework/node/resource.go`. see https://github.com/kubernetes/kubernetes/pull/79879#discussion_r301917574
Thanks!